pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -472,7 +472,7 @@ func newKubeVirtConfigForCR(cr *hcov1alpha1.HyperConverged, namespace string) *c
 		Data: map[string]string{
 			"feature-gates":       "DataVolumes,SRIOV,LiveMigration,CPUManager,CPUNodeDiscovery,Sidecar",
 			"migrations":          `{"nodeDrainTaintKey" : "node.kubernetes.io/unschedulable"}`,
-			"selinuxLauncherType": "spc_t",
+			"selinuxLauncherType": "virt_launcher.process",
```

Comment:
Stu, do we also have to update it on existing deployments upgraded to this code?
Please see also: https://github.com/kubevirt/hyperconverged-cluster-operator/pull/577

Reply:
Yes we would. What does that entail?

Reply:
We don't want to reconcile anymore the whole configMap to preserve user set values (on feature-gates for instance) so we have to know which keys should be reconciled on upgrades.